### PR TITLE
Moving tilt to use helm instead of static yaml

### PR DESCRIPTION
**Pull request by networkop**
TODO:
- [x] remove old `test/kubernetes.yaml`
- [x] check if ingress-nginx can be changed to helm
- [x] pin versions on all helm charts
- [x] rebase
- [x] README updates

### CODE REVIEW

**Comment by networkop** [comment]
@Kaezon can you give a quick overview? I wanted to merge this before I forget all about this PR.

**Comment by Kaezon** [comment]
Yes, sorry. I let this fall off my plate. I'll go back over it now.

**Review by Kaezon — CHANGES_REQUESTED** [review]
The only thing I see is that one note I made on the VirtualServer CR.
Pretty sure you can just point it to one of the services defined in `test.yaml` and call it good.

File: test/nginxinc-kubernetes-ingress/resources.yaml [hunk 1]
```diff
@@ -47,79 +28,3 @@ spec:
...
-    spec:
-      containers:
-      - image: nginx
-        imagePullPolicy: Always
-        name: nginx
----
-apiVersion: v1
-kind: Service
-metadata:
-  name: nginx-test-a
```
[the remark this answers is not in the document]
Comment (Kaezon) [reply test/nginxinc-kubernetes-ingress/resources.yaml:76]: Also, I thought this note was visible when I made it almost a week ago, but I guess it only shows up when I complete the review?

[the remark this answers is not in the document]
Comment (networkop) [reply test/nginxinc-kubernetes-ingress/resources.yaml:76]: yep, I see. I'll probably factor out that deployment+service part and make to make them common for all Ingress/VR resources.

**Pull request merged**